derive/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
```

Comment:
Was this accidentally added?

Reply:
Yeah, thought this was wiped out. Must have snuck back in a rogue rebase. I'll delete this.